src/app/components/datatable/datatable.ts

```diff
@@ -2348,8 +2348,23 @@ export class DataTable implements AfterViewChecked,AfterViewInit,AfterContentIni
             this.updateDataToRender(this.value);    
     }
     
+    getSelection() {
+      return this._selection;
+    }
+    
+    public exportSelectedRowsToCSV() {
```

Comment:
The way this method works differs from original `exportCSV` when there are no data. `exportCSV` would still make browser download and empty CSV file containing only header. While your version silently does nothing when nothing is selected.
It would be nice to align the two IMO.